client/webserver/site/src/html/markets.tmpl

```diff
@@ -116,6 +116,21 @@
             </div>
 
             <div class="fs15 pt-3 text-center d-hide" id="loaderMsg"></div>
+            
+            {{- /* REGISTRATION STATUS */ -}}
+            <div class="d-hide" id="registrationStatus">
+              <div class="p-0 w-100">
+                <div class="d-flex flex-column justify-content-center align-items-center">
+                  <span id="regStatusTitle" class="title"></span>
+                  <p>In order to trade at <span id="regStatusDex"></span>, the registration fee payment
+                    needs <span id="confReq"></span> confirmations. The number of confirmations received will appear below once it
+                    is available.
+                  </p>
```

Comment:
~~The number of confirmations received will appear below once it is available.`~~. 

It's always available now, I think.